tests/stub/bookmarks/scripts/v3/send_and_receive_bookmark_read_tx.script

```diff
@@ -10,7 +10,5 @@ S: SUCCESS {"fields": ["name"]}
    SUCCESS {}
 C: COMMIT
 S: SUCCESS {"bookmark": "neo4j:bookmark:v1:tx4242"}
-{?
-    C: GOODBYE
-    S: <EXIT>
-?}
+*: RESET
```

Comment:
Why not `?:  RESET`?

Reply:
Because `!: AUTO RESET` would've allowed for any number of `RESET`s. After all, we're not testing if the driver does it's job efficiently by only sending the least number of needed `RESET`s. Any number will work even though it's not optimized.